eval_engines/ngspice/ngspice_wrapper.py

```
@@ -23,11 +28,19 @@ def __init__(self, num_process, yaml_path, path, root_dir=None):
         else:
             self.root_dir = root_dir
 
-        with open(yaml_path, 'r') as f:
+        with open(yaml_path, "r") as f:
             yaml_data = yaml.load(f)
```

Ruff (0.12.2)

Probable use of unsafe `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)